pkg/cmd/issue/view/view.go

```diff
@@ -127,35 +127,45 @@ func viewRun(opts *ViewOptions) error {
 
 		projectsV1Support := opts.Detector.ProjectsV1()
 		if projectsV1Support == gh.ProjectsV1Supported {
-			lookupFields.Add("projectCards")
+			fields.Add("projectCards")
 		}
 	}
 
 	opts.IO.DetectTerminalTheme()
 
 	opts.IO.StartProgressIndicator()
-	lookupFields.Add("id")
+	defer opts.IO.StopProgressIndicator()
 
-	issue, err := issueShared.FindIssueOrPR(httpClient, baseRepo, opts.IssueNumber, lookupFields.ToSlice())
+	fields.Add("id")
+
+	issue, err := issueShared.FindIssueOrPR(httpClient, baseRepo, opts.IssueNumber, fields.ToSlice())
 	if err != nil {
-		return err
+		var partialLoadErr *issueShared.PartialLoadError
+		if opts.Exporter == nil && errors.As(err, &partialLoadErr) {
+			fmt.Fprintf(opts.IO.ErrOut, "warning: %s\n", partialLoadErr.Error())
+		} else {
+			return err
+		}
 	}
```

Comment:
Currently, as of recent changes, now we do not treat partial errors differently, and just bail out if error is not `nil`. I don't think bringing back the old behaviour is the right decision here. So, let's just return the `err` and avoid dealing with partial errors. Also, it's not a well-tested path and it's better to leave it as it was.